src/jmcomic/jm_plugin.py

```
@@ -383,6 +383,168 @@ def get_zip_path(self, album, photo, filename_rule, suffix, zip_dir):
+    @staticmethod
+    def unified_path(f):
+        return fix_filepath(f, os.path.isdir(f))
+
+    def zip_album(self, album, photo_dict: dict, zip_path, path_to_delete):
+        """
+        压缩album文件夹
+        """
+
+        album_dir = self.option.dir_rule.decide_album_root_dir(album)
+            
+        with pyzipper.AESZipFile(zip_path, 'w', pyzipper.ZIP_DEFLATED) as f:
```

Ruff (0.8.2)

`pyzipper` may be undefined, or defined from star imports

(F405)